packages/cli/src/make.js

```diff
@@ -68,7 +69,7 @@ export const makeCommand = async ({
       importPath !== undefined
         ? E(party).importUnsafeAndEndow(
             workerName,
-            path.resolve(importPath),
+            url.pathToFileURL(path.resolve(importPath)).href,
```

Comment:
Does this change behavior? Do any of the tests show how?

Reply:
No behavior change, but if this were released, it would be a breaking change since the CLI and daemon would have to be updated together for it to continue working. This also will eventually require an `endo reset` since it changes the formula schema.

This is a coördinated change between the CLI and the daemon, such that the daemon is portable between web and node, both of which can dynamically import URLs. The CLI does the path to URL conversion instead of the daemon.

Reply:
> The CLI does the path to URL conversion instead of the daemon.

hm. That sounds like a behavior change to me.